lib/ansible/modules/network/f5/bigip_static_route.py

```diff
@@ -286,22 +295,23 @@ def destination_ip(self):
     def netmask(self):
         if self._values['netmask'] is None:
             return None
-        # Check if numeric
+        result = -1
         try:
             result = int(self._values['netmask'])
-            if 0 <= result < 256:
-                return result
+            if 0 < result < 256:
+                pass
```

Comment:
This doesn't make sense: you don't need if-clause if you're not going to do anything in the block.